### ISSUE TITLE
Error in lightning_workflow.ipynb

### ISSUE THREAD

**Issue by robmarkcole** [azavea/raster-vision#2213]
At the cell:
```python
from rastervision.core.data import SemanticSegmentationLabels

model.eval()
predictions = get_predictions(val_dl)
pred_labels = SemanticSegmentationLabels.from_predictions(
    val_ds.windows,
    predictions,
    smooth=True,
    extent=val_ds.scene.extent,
    num_classes=len(class_config),
)
scores = pred_labels.get_score_arr(pred_labels.extent)

---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
Cell In[24], line 5
      3 model.eval()
      4 predictions = get_predictions(val_dl)
----> 5 pred_labels = SemanticSegmentationLabels.from_predictions(
      6     val_ds.windows,
      7     predictions,
      8     smooth=True,
      9     extent=val_ds.scene.extent,
     10     num_classes=len(class_config),
     11 )
     12 scores = pred_labels.get_score_arr(pred_labels.extent)

File /home/zeus/miniconda3/envs/cloudspace/lib/python3.10/site-packages/rastervision/core/data/label/semantic_segmentation_labels.py:220, in SemanticSegmentationLabels.from_predictions(cls, windows, predictions, extent, num_classes, smooth, crop_sz)
    194 """Instantiate from windows and their corresponding predictions.
    195 
    196 Args:
   (...)
    217     Otherwise, a SemanticSegmentationDiscreteLabels.
    218 """
    219 labels = cls.make_empty(extent, num_classes, smooth=smooth)
--> 220 labels.add_predictions(windows, predictions, crop_sz=crop_sz)
    221 return labels
...
    499 dst_yslice, dst_xslice = window_dst.to_slices()
--> 500 self.pixel_scores[..., dst_yslice, dst_xslice] += pixel_class_scores
    501 self.pixel_hits[dst_yslice, dst_xslice] += 1

ValueError: operands could not be broadcast together with shapes (2,325,325) (3,325,325) (2,325,325) 
```

**Comment by AdeelH** [comment]
Thanks for catching that! This can be fixed by changing
```py
pred_labels = SemanticSegmentationLabels.from_predictions(
    ...
    num_classes=len(class_config),
)
```
to
```py
pred_labels = SemanticSegmentationLabels.from_predictions(
    ...
    num_classes=len(class_config) + 1,
)
```
The error is due to a discrepancy between the `ClassConfig` defined in the notebook and the `ClassConfig` the model being loaded was trained with. The latter has an additional `"null"` class (for NODATA pixels). I plan to fix this by removing that class from the loaded model.

**Comment by robmarkcole** [comment]
Excellent, can confirm that resolves the issue

**Comment by AdeelH** [comment]
Fixed in #2215
